tox.ini

```diff
@@ -65,7 +65,7 @@ commands =
 
 [testenv:docs]
 description = build documentations
-basepython = python3.8
+basepython = python3.9
```

Comment:
Any reason we can't make it 3.10? :thinking:

Reply:
We should probably keep it synced with RTD, dunno if they support 3.10 yet.

Reply:
I see :thinking: